### PR TITLE
Optimize the FAQ TOC

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

Under existing FAQs, the scope of the [Deploy and Maintain FAQs](https://docs.pingcap.com/tidb/stable/deploy-and-maintain-faq) is too large and [Upgrade FAQs](https://docs.pingcap.com/tidb/stable/upgrade-faq) is not included. 

To solve this problem and also align the FAQs TOC with the existing docs, This PR split the Cluster management FAQ and Monitoring FAQ from Deploy and Maintain FAQs. 

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM

File: faq/deploy-and-maintain-faq.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: 部署运维 FAQ
-summary: 介绍 TiDB 集群运维部署的常见问题、原因及解决方法。
+title: 安装部署 FAQ
+summary: 介绍 TiDB 集群安装部署的常见问题、原因及解决方法。
 aliases: ['/docs-cn/dev/faq/deploy-and-maintain-faq/']
 ---
 
 # 部署运维 FAQ
```
Comment (lilin90) [inline faq/deploy-and-maintain-faq.md:7]:
```suggestion
# 安装部署 FAQ
```

File: faq/manage-cluster-faq.md [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+本文介绍管理 TiDB 集群时的常见问题、原因及解决方法。
+
+## 集群日常管理
+
+### TiDB 如何登录？
+
+和 MySQL 登录方式一样，可以按照下面例子进行登录。
+
+`mysql -h 127.0.0.1 -uroot -P4000`
```
Comment (lilin90) [inline faq/manage-cluster-faq.md:16]: How about making it a code block?
````suggestion
{{< copyable "shell-regular" >}}

```shell
mysql -h 127.0.0.1 -uroot -P4000
```
````

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**